### PR TITLE
Connect database

### PR DESCRIPTION
**Pull request by lennart-kaminsky**
[US - Connect Database](https://github.com/users/GoekhanAgkurt/projects/2/views/1?pane=issue&itemId=45919725)

### Also in this PR:

- LottieFile Component
- 404 error page of Next.js

### CODE REVIEW

File: pages/api/products/[id].js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import dbConnect from "@/db/connect";
+import Product from "@/db/models/Product";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  //   if (!id) {
+  //     return;
+  //   }
```
Comment (Roland-Hufnagel) [inline pages/api/products/[id].js:10]: you don't want to show this 😀

File: pages/api/stores/[id].js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import dbConnect from "@/db/connect";
+import Store from "@/db/models/Store";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  // if (!id) {
+  //   return;
+  // }
```
Comment (Roland-Hufnagel) [inline pages/api/stores/[id].js:10]: the same

**Review by Roland-Hufnagel — COMMENTED** [review]
Hey Grocery Team. That's a really big update. Only found some minor important things. Great Work 💪

**Review by Roland-Hufnagel — APPROVED** [review]

**Comment by lennart-kaminsky** [comment]
> Hey Grocery Team. That's a really big update. Only found some minor important things. Great Work 💪

Hey @Roland-Hufnagel, thanks for taking the time to review this big PR! ❤️  We changed/removed the things you were mentioning✌️

**Pull request merged**